gettsim/data/arbeitsl_geld_2.yaml

```diff
@@ -1,730 +1,274 @@
-a2we:
-  name:
-    de: Abzugsfähige Werbungskostenpauschale
-    en: Allowance for expenses
-  description:
-    de: Abzugsfähige monatliche Werbungskostenpauschale. § 6 (1) Nr. 3 a) ALG II-V. 2005-01.01 bis 2016-07-31. Betrag mit V. v. 26.07.2016 BGBl. I S. 1858 entfallen.
-    en:
-  values:
-    2005-01-01:
-      value: 15.33
-      note: V. v. 20.10.2004 BGBl. I S. 2622. Es gilt 1/60 der ein Sechzigstel der steuerrechtlichen Werbungskostenpauschale (Parameter werbung). Berechnung werbung/60 = 920/60.
-    2011-01-01:
-      value: 15.33
-      note: V. v. 21.06.2011 BGBl. I S. 1175. Fester Betrag. Betrag unverändert.
-a2zu1:
-  name:
-    de: Befristeter Zuschlag nach Bezug von Arbeitslosengeld (2005-2010)
-    en: Temporary additional benefit after UI (2005-2010)
-  description:
-    de: Befristeter Zuschlag nach Bezug von Arbeitslosengeld ("Armutsgewöhnungszuschlag"). Bis einschließlich Dezember 2010 bestand für ehemalige Arbeitslosengeldempfänger die Möglichkeit einen Zuschuss von bis zu 160 Euro (320 für Verheiratete) pro Monat für die Dauer von maximal zwei Jahren zu beziehen. § 24 SGB II. Aufgehoben zum 01.01.2011 mit G. v. 09.12.2010 BGBl. I S. 1885.
-    en:
-  values:
-    2005-01-01:
-      value: 160
-      note: G. v. 24.12.2003 BGBl. I S. 2954.
-a2_but_1:
-  name:
-    de: Leistungen für Bildung und Teilhabe, Schulbedarf
-    en: Benefits for education and participation, school supplies
-  description:
-    de: Leistungen für Bildung und Teilhabe, Persönlicher Schulbedarf, Durchschnittsbetrag pro Schulkind und Monat. §28 (2) SGB II.
-    en:
-  values:
-    2011-01-01:
-      value: 8.33
-      note: G. v. 24.03.2011 BGBl. I S. 453. Berechnung 70+30/12.
-a2_but_2:
-  name:
-    de: Leistungen für Bildung und Teilhabe, Teilhabe am sozialen Leben
-    en: Benefits for education and participation, social activities
-  description:
-    de: Leistungen für Bildung und Teilhabe, Teilhabe am sozialen Leben, Betrag pro Kind und Monat. §28 (7) SGB II.
-    en:
-  values:
-    2011-01-01:
-      value: 10
-      note: G. v. 24.03.2011 BGBl. I S. 453.
-    2019-08-01:
-      value: 15
-      note: G. v. 29.04.2019 BGBl. I S. 530.
+        reference: G. v. 14.08.2005 BGBl. I S. 2407.
+        deviation_from: arbeitsl_geld_2.e_anr_frei
+        2:
+            upper_threshold: 1500
+
+regelsatz:
+  name:
+    de: Regelsatz, seit Beträge differenzieren sich nach Regelbedarfsstufen
+    en: Standard rate, since 2011 amounts differ by "Regelbedarfsstufen"
+  description:
+    de: § 20 V SGB II. Für 2005 bis 2010 siehe Bekanntmachungen zu § 20. Seit 2011 wird der Regelbedarf durch das Regelbedarfs-Ermittlungsgesetz (RBEG) ermittelt. Neufassung SGB II § 20 (1a) und (2) durch Artikel 6 G. v. 22.12.2016 BGBl. I S. 3159.
+    en:
+  note:
+    de: Regelbedafstufen (1 bis 6); 1, Alleinstehender Erwachsener; 2, Erwachsene in Partnerschaft; 3, Erwachsene nicht in Partnerschaft; 4, Kinder 14 bis 18; 5, Kinder 6 bis 15; 6, Kinder 0 bis 5
```

Comment:
Is this the final style of the note/lang key here? You could also have multiple keys `de_1, de_2,... de_6` like for `vermögensfreibetrag` below (I'm not sure I like that `_1` numbering). Anyway the meaning is not too precise here as the scheme changes 2011-01-01.

Reply:
We explain above that this rule starts in 2011. I am not sure about this numbering as well. I just introduced to have some kind of possibility for multi lines comment. I think a better way is to start at some point a kind of database in the documentation, where we can explain parameters in more detail!